### ISSUE TITLE
Compiling a subgraph from a graph  in keras

### ISSUE THREAD

**Issue by bienbinod** [fchollet/keras#1798]
Is it possible to compile a subgraph at a time using keras? For example, in multitask setup, where we minimize the loss of multiple tasks.  Diagram taken from the issue #1441 
![e77736d2-b703-11e5-94f9-a561203ae357](https://cloud.githubusercontent.com/assets/11754701/13259059/10fba13a-da56-11e5-91b8-7a14651d4f11.png)

Can we pass training example from lang1 only and update the parameters relevant for the lang 1 and at another time for the lang2 and so on?

**Comment by neggert** [comment]
You can probably accomplish what you want by copying weights around.

**Comment by bienbinod** [comment]
@neggert Thanks, this is what I thought also. But, is not it possible to specify specific path to compile in graph ? @fchollet

**Comment by fchollet** [comment]
Simplest thing is to copy the weights. Otherwise you can always define your own `K` function based on arbitrary inputs / outputs in the graph.
